backend/tracim_backend/lib/core/user.py

```diff
@@ -477,15 +478,18 @@ def _ldap_authenticate(
             #             )
             #         )
             name = None
+            mail = None
+            username = None
             if self._config.LDAP_NAME_ATTRIBUTE:
                 name = ldap_data[self._config.LDAP_NAME_ATTRIBUTE][0]
+            if self._config.LDAP_MAIL_ATTRIBUTE:
+                mail = ldap_data[self._config.LDAP_MAIL_ATTRIBUTE][0]
+            if self._config.LDAP_USERNAME_ATTRIBUTE:
+                username = ldap_data[self._config.LDAP_USERNAME_ATTRIBUTE][0]
```

Comment:
In the event there is several entries with the same key, matching in the array should be performed first, and [0] should only be a fallback.

However this is not standard to LDAP so it should be trated later. An issue should be made

Reply:
I'm making the issue

Reply:
https://github.com/tracim/tracim/issues/6178

Reply:
#6179